src/Downloader.ts

```diff
@@ -241,26 +227,29 @@ class Downloader {
     }
   }
 
-  public async checkCapabilities(testArticleId: string='MediaWiki:Sidebar'): Promise<void> {
-
-    // By default check all API's responses and set the capabilities
-    // accordingly. We need to set a default page (always there because
-    // installed per default) to request the REST API, otherwise it would
-    // fail the check.
-    this.mwCapabilities.mobileRestApiAvailable = await this.checkApiAvailabilty(this.mw.getMobileRestApiArticleUrl(testArticleId));
-    this.mwCapabilities.desktopRestApiAvailable = await this.checkApiAvailabilty(this.mw.getDesktopRestApiArticleUrl(testArticleId));
-    this.mwCapabilities.veApiAvailable = await this.checkApiAvailabilty(this.mw.getVeApiArticleUrl(testArticleId));
-
-    // Coordinate fetching
+  public async checkCoordinateAvailablity(): Promise<boolean> {
     const reqOpts = objToQueryString({
       ...this.getArticleQueryOpts(),
     });
     const resp = await this.getJSON<MwApiResponse>(`${this.mw.apiUrl.href}${reqOpts}`);
     const isCoordinateWarning = resp.warnings && resp.warnings.query && (resp.warnings.query['*'] || '').includes('coordinates');
     if (isCoordinateWarning) {
       logger.info(`Coordinates not available on this wiki`);
-      this.mwCapabilities.coordinatesAvailable = false;
+      return false;
```

Comment:
@kelson42 hasCoordinateApi is declared in MediaWiki this is there because to get coordinates we required a specific kind JSON request and this is needed for that purpose